### ISSUE TITLE
Can't override `--config-file` in `aliases`

### ISSUE THREAD

**Issue by hundredwatt** [basecamp/kamal#1027]
I have a secondary service that's deployed from my app's repo via a separate configuration file. 

I can deploy the secondary service with `kamal deploy --config-file=config/other.deploy.yml` and that command succeeds.

I tried to make this command an alias in my main app's `config/deploy.yml`, but I get a registry error because the secondary service uses a different registry but kamal tries to use the main app's registry to pull the image:

```
docker stderr: Unable to find image main.myregistry.com/secondary:latest' locally
```

When in `config/other.deploy.yml`, I have the `registry.server` set to `other.myregistry.com`.

My alias configuration in `config/deploy.yml` looks like:

```yaml
aliases:
  "other-deploy": "deploy --config-file=config/other.deploy.yml"
```

If I'm using kamal in an unintended way please tell me 😅. But it would be nice if I could configure an alias to deploy my secondary service so I don't have to add `--config-file=config/other.deploy.yml` every time.

**Comment by djmb** [comment]
Yes that's not something that was supported, but making it work turned out to be easy enough - https://github.com/basecamp/kamal/pull/1032
